### PR TITLE
Fix release automation

### PR DESCRIPTION
**Pull request by t0yv0**
Editing the GitHub actions workflow to propagate new bridge releases to all providers.

Fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/1470
Fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/1474

### CODE REVIEW

**Comment by t0yv0** [comment]
Example trigger receipt: https://github.com/pulumi/pulumi-aiven/actions/runs/6634481479/job/18023972713

**Comment by t0yv0** [comment]
Example PR: https://github.com/pulumi/pulumi-mailgun/pull/208

**Review by iwahbe — APPROVED** [review]
This is fine as it stands, but I would prefer that we pull the matrix from `ci-mgmt`. Ideally, we wouldn't maintain multiple lists of all our providers.

File: .github/workflows/update-providers-auto.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          # Not specifying target-bridge-version in the payload will make it upgrade to the latest.
+          client-payload: |-
+            {
+               "pr-reviewers": "t0yv0",
+               "automerge": true
+            }
+    strategy:
+      fail-fast: false
+      matrix:
+        provider:
```
Comment (iwahbe) [inline .github/workflows/update-providers-auto.yml:32]: This matrix is missing pulumi-oci. I really think we should draw this list from ci-mgmt.

**Comment by t0yv0** [comment]
You are right. Scripted drawing from ci-mgmt. Test-driving.

**Pull request merged**